commands/CreateSuite.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * @adonisjs/assembler
+ *
+ * (c) AdonisJS
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import { BaseCommand, args, flags } from '@adonisjs/core/build/standalone'
+import { files, logger } from '@adonisjs/sink'
+import { join } from 'path'
+
+/**
+ * Create a new test suite
+ */
+export default class CreateSuite extends BaseCommand {
+  public static commandName = 'create-test-suite'
+  public static description = 'Create a new test suite'
+
+  /**
+   * Name of the test suite to be created
+   */
+  @args.string({ description: 'Name of the test suite' })
+  public suiteName: string
+
+  /**
+   * Should add a sample test file
+   */
+  @flags.boolean({ description: 'Add a sample test file' })
+  public addSampleTestFile: boolean = true
```

Comment:
This can renamed to `withExampleTest` and can be mentioned as `--with-example-test`